brainglobe_utils/IO/image/load.py

```diff
@@ -768,19 +768,39 @@ def read_z_stack(path):
 def read_with_dask(path):
     """
     Based on https://github.com/tlambert03/napari-ndtiffs
-    :param path:
-    :return:
+    Reads a folder of tiffs lazily.
+
+    Note that it will make tifffile.imread ignore OME metadata,
+    because this can cause issues with correct metadata reading.
+    See https://forum.image.sc/t/tifffile-opening-individual-ome-tiff-files-as-single-huge-array-even-when-isolated/77701
+
+    :param path: folder with tifs.
+    :return: dask array containing stack of tifs
     """
     path = str(path)
     if path.endswith(".txt"):
         with open(path, "r") as f:
             filenames = [line.rstrip() for line in f.readlines()]
 
     else:
-        filenames = glob.glob(os.path.join(path, "*.tif"))
```

Comment:
I might be wrong, but would the following syntax achieve the equivalent?

```python
filenames = glob.glob(os.path.join(path, "*.tif")) or glob.glob(os.path.join(path, "*.tiff"))
if not filenames:
    raise ValueError(f"Folder {path} does not contain any .tif or .tiff files")
```
If yes, should we still prefere the verbose version for readability or nor?

Reply:
Good point - I wanted to do something like this but couldn't think about how!